File: Frontend/src/app/models/PicturesData.ts
import { HostingObject } from "./HostingObject";

export class PicturesData {
    id: number;
    pictures: string;
    hostingObject: HostingObject;

    constructor(id: number, url: string, hostingObjectId: HostingObject) {
        this.id = id;
        this.pictures = url;
        this.hostingObject = hostingObjectId;
    }
}

File: Backend/NaissusEvents/Models/PicturesData.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;



namespace Models
{
    
    public class PicturesData
    {
        [Key]
        public int Id { get; set; }
        public string Pictures { get; set; }
        public HostingObject hostingObject {get;set;}


    }
}
